pkg/sdn/api/v1/types.go

```diff
@@ -20,6 +20,8 @@ type ClusterNetwork struct {
 
 	// Network is a CIDR string specifying the global overlay network's L3 space
 	Network string `json:"network" protobuf:"bytes,2,opt,name=network"`
+	// PLACEHOLDER
+	ClusterDef []ClusterNetworkEntry `json:"clusterDef" protobuf:"bytes,6,rep,name=clusterDef"`
```

Comment:
https://github.com/kubernetes/community/blob/master/contributors/devel/api_changes.md has some information on changing kubernetes API objects, and in particular discusses the backward-compatibility snafus that come with trying to turn a single-element field into an array... (The "API Conventions" link near the top of that document is also filled with useful information about how kubernetes APIs work.)

The easiest way to make this work backward-compatibly would be to say that the "default"/first network is still specified by the `Network` and `HostSubnetLength` fields, but the 2nd to Nth networks are part of a new array field (which could be called `AdditionalNetworks` or something like that).

You need a doc comment above the field, and between the doc comment and the actual declaration should be

    // +optional

to indicate that it's an optional field